src/chart/generateCategoricalChart.tsx

```diff
@@ -662,16 +664,26 @@ const getAxisNameByLayout = (layout: LayoutType) => {
  * @return {Object} The offset of main part in the svg element
```

Comment:
Can we correct the JSDoc too since its there?

Reply:
Updated